packages/core-modules/contracts/interfaces/IOwnerModule.sol

```diff
@@ -0,0 +1,8 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IOwnerModule {
```

Comment:
Should we be consistent and add interfaces to all modules? I believe the UpgradeModule is the only one without an interface right now.